showcase/webpack.config.js

```diff
@@ -25,10 +25,11 @@ const commonConfig = {
   mode: IS_PROD
     ? 'production'
     : 'development',
-  
+
   entry: {
     app: './src/index.js',
-    sw: './src/sw.js'
+    sw: './src/sw.js',
+    extra: '../elm-antd-extras'
```

Comment:
Yup, this works fine for the showcase! Although I am wondering if there's a way to specify that the `extra` bundle be injected into the `head` of the html instead of the bottom of the `body` ... but that's not a big deal right now.